osu.Game/Screens/OnlinePlay/Multiplayer/GameplayChatDisplay.cs

```diff
@@ -37,14 +38,17 @@ public GameplayChatDisplay(Room room)
             : base(room, leaveChannelOnDispose: false)
         {
             RelativeSizeAxes = Axes.X;
-
             Background.Alpha = 0.2f;
+        }
 
-            TextBox.PlaceholderText = ChatStrings.InGameInputPlaceholder;
+        [BackgroundDependencyLoader]
+        private void load(OsuConfigManager config)
+        {
+            TextBox.PlaceholderText = ChatStrings.InGameInputPlaceholder(config.LookupKeyBindings(GlobalAction.ToggleChatFocus));
```

Comment:
Curious what @ppy/team-client opinions are on *calling* a `Func<>` property... I think it's best we don't do this? and maybe even make it `set`-only?

Also of note, this won't update immediately when the binding it changed, but I guess that's alright.

Reply:
Have made some changes to this flow in https://github.com/peppy/osu/commit/4b038c37627bcabce7b299f8d60995dbb455be40